### PR TITLE
Switching tabletmanager to the new tabletserver API.

### PR DESCRIPTION
**Pull request by alainjobart**
@sougou 
After this, tabletmanager doesn't use 'StartService' any more, I'll let you clean it up.

### CODE REVIEW

**Comment by sougou** [comment]
lgtm

**Comment by alainjobart** [comment]
@sougou PTAL this is ready to merge now, when Travis passes.

**Comment by sougou** [comment]
lgtm

File: go/vt/tabletmanager/after_action.go [hunk 1]
```diff
@@ -141,9 +116,38 @@ func (agent *ActionAgent) loadKeyspaceAndBlacklistRules(tablet *pbt.Tablet, blac
...
+		stats.SecondsBehindMasterFilteredReplication, stats.BinlogPlayersCount = agent.BinlogPlayerMap.StatusSummary()
+	}
+	if healthError != nil {
+		stats.HealthError = healthError.Error()
+	}
+	var ts int64
+	if !terTime.IsZero() {
+		ts = terTime.Unix()
+	}
+	defer func() {
```
Comment (enisoc) [inline go/vt/tabletmanager/after_action.go]: Why is this a `defer`? Was it meant to be `go`?

Comment (alainjobart) [inline go/vt/tabletmanager/after_action.go]: I think it was yes...

**Pull request merged**